special-use/special-use-suds-mapping.json

```diff
@@ -6,97 +6,97 @@
-      "location": {
-        "type": "object",
-        "properties": {
-          "latitude": { "type": "integer", "suds-application": "Contact", "suds-field": "International Code"},
-          "longitude": { "type": "integer", "suds-application": "Contact", "suds-field": "Area Code"},
-        },
-        "required": ["phoneAreaCode","phoneNumber","phoneType"]
-      },
       "non-commercial-fields": {
         "type": "object",
         "properties": {
-      		"activityDescription": { "type": "string", "suds-application": "Application", "suds-field" : "Purpose"},
-      		"locationDescription": {	"type": "string", "suds-table": "application", "suds-field" : "purpose"},
-          "startDateTime": { "type": "dateTime", "suds-table": "application", "suds-field" : "remarks"},
-          "endDateTime": { "type": "dateTime", "suds-table": "application", "suds-field" : "remarks" },
-          "numberParticipants" : { "type": "integer",  "suds-field" : "remarks" }
+      		"activityDescription": { "type": "string", "note": "See the 'not accessible to middleware' note."},
+      		"locationDescription": {	"type": "string", "note": "See the 'not accessible to middleware' note."},
+          "startDateTime": { "type": "dateTime", "note": "See the 'not accessible to middleware' note."},
+          "endDateTime": { "type": "dateTime", "note": "See the 'not accessible to middleware' note." },
+          "numberParticipants" : { "type": "integer",  "note": "See the 'not accessible to middleware' note." },
+          "NOT_ACCESSIBLE_VIA_PUBLIC_API": { "basic-api-field": "PURPOSE", "note" : "This field should be a concatenation of activityDescription, locationDescription, startDateTime, endDateTime."},
+          "NOT_ACCESSIBLE_VIA_PUBLIC_API": { "basic-api-field": "CR_EXEMPT", "value" : "Y"},
+          "NOT_ACCESSIBLE_VIA_PUBLIC_API": { "basic-api-field": "FORM_NAME", "value" : "FS-2700-3b"},
           },
-        "pre-determined-suds-fields":
-        {
-          "!COMMENT" : "If the permit type is non-commercial, these fields can be set in SUDS without any additional input from the API user.",
-          "crExempt" : { "type" : "string", "suds-application" : "Application", "suds-field" : "value", "no"},
-          "code" : { "type" : "integer", "suds-application" : "Application", "suds-field" : "value", "191"}
-        }
       },
       "temp-outfitter-fields": {
         "type": "object",
         "properties": {
       		"insuranceCertificate": {	"type": "file" },
           "goodStandingEvidence": {
       			"description": "Certificate of good standing, operating agreement, or other evidence the business is in good standing",
+            "note": "See the 'not accessible to middleware' note below this one.",
             "type": "file"
           },
           "operatingPlan": {
       			"description": "A completed Word or RTF operating plan template.",
+            "note": "See the 'not accessible to middleware' note below this one.",
```

Comment:
Please change to `autopopulated by middleware not intake module." per our discussion.